datajoint/autopopulate.py

```diff
@@ -222,7 +222,11 @@ def handler(signum, frame):
             make_kwargs=make_kwargs,
         )
 
-        if processes == 1:
+        if processes < 0:
+            raise Exception("processes must not be negative")
```

Comment:
Root `Exception` should be avoided. A more specific exception should be used. Why is the check for negativity necessary?

Reply:
I will add a more specific exception here. The reason I included this to begin with is as a simple input validation. If a user tries to set processes to a negative number, something has gone wrong, and raising an exception should allow them to handle it.

Reply:
I specified the exception to be a value error.